custom_components/kia_uvo/sensor.py

```diff
@@ -48,7 +51,8 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     INSTRUMENTS.append(("geocodedLocation", "Geocoded Location", "vehicleLocation.geocodedLocation.display_name", None, "mdi:map", None))
     INSTRUMENTS.append(("carBattery", "Car Battery", "vehicleStatus.battery.batSoc", PERCENTAGE, "mdi:car-battery", DEVICE_CLASS_BATTERY))
 
-    INSTRUMENTS.append(("temperatureSetpoint", "Set Temperature", "vehicleStatus.airTemp.value", TEMP_CELSIUS, None, DEVICE_CLASS_TEMPERATURE))
+    if config_entry.data.get(CONF_REGION, DEFAULT_REGION) is not REGION_USA:
```

Comment:
Are you able to confirm if this reports correctly when the car is running?  If so it may be the same as EU and CA. Would rather fix it for all regions if that is the case.